Chapter-4-applying-tactical-domain-driven-design/Fitnet.Contracts/Src/Fitnet.Contracts.Core/TerminateContract/BusinessRules/TerminationIsNotPossibleUntilThreeMonthsHaveElapsedRule.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace EvolutionaryArchitecture.Fitnet.Contracts.Core.TerminateContract.BusinessRules;
+
+using EvolutionaryArchitecture.Fitnet.Common.Core.BusinessRules;
+
+internal sealed class TerminationIsNotPossibleUntilThreeMonthsHaveElapsedRule(DateTimeOffset bindingFrom, DateTimeOffset terminatedAt)
```

Comment:
What about positive approach? I mean to use:

TerminationIsPossibleOnlyAfterThreeMonthsHavePassed

Reply:
Yeah in this way is simpler :D I will change it